include/dxc/DXIL/DxilShaderFlags.h

```diff
@@ -160,7 +168,7 @@ class ShaderFlags {
   void SetDerivativesInMeshAndAmpShaders(bool flag) {
     m_bDerivativesInMeshAndAmpShaders = flag;
   }
-  bool GetDerivativesInMeshAndAmpShaders() {
+  bool GetDerivativesInMeshAndAmpShaders() const {
```

Comment:
This is completely unrelated. Including unrelated changes makes it harder to review.

Reply:
I'm working on separating this change out.

Reply:
Change removed from this PR.